addons/events.py

```diff
@@ -77,6 +77,9 @@ def __init__(self, bot):
         'fleashop',
         'ciangle',
         'fieashop',
+        'frshp',
+        'free****',
+        'free*shop*', #and others like it..
```

Comment:
Huh... thats new to me probably because I tend not to leave them for cleaner code